spec/versionedsignedbeaconblock.go

```diff
@@ -1043,6 +1043,49 @@ func (v *VersionedSignedBeaconBlock) ExecutionRequests() (*electra.ExecutionRequ
 	}
 }
 
+// ExecutionPayload returns the execution payload of the beacon block.
```

Comment:
```suggestion
// ExecutionPayload returns the execution payload of the signed beacon block.
```

Reply:
that's not fully right here,  executiuon payload is part of the BeaconBlock, which can be shared in a signed or unsigned way ;)